### PR TITLE
net.minecraft.client.gui.screens.inventory.*

### PR DESCRIPTION
**Pull request by MelanX**
PR mentioned in #78

### CODE REVIEW

File: data/net/minecraft/client/gui/screens/inventory/AbstractCommandBlockEditScreen.mapping [hunk 1]
```diff
@@ -3,6 +3,13 @@ CLASS net/minecraft/client/gui/screens/inventory/AbstractCommandBlockEditScreen
 		ARG 1 keyCode
 		ARG 2 scanCode
 		ARG 3 modifiers
+	METHOD lambda$init$0 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 button
+	METHOD lambda$init$1 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 button
+	METHOD lambda$init$2 (Lnet/minecraft/client/gui/components/CycleButton;Ljava/lang/Boolean;)V
+		ARG 1 cycleButton
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/AbstractCommandBlockEditScreen.mapping]:
```suggestion
		ARG 1 outputButton
```
(or just `button`)

File: data/net/minecraft/client/gui/screens/inventory/AbstractFurnaceScreen.mapping [hunk 2]
```diff
@@ -18,6 +18,8 @@ CLASS net/minecraft/client/gui/screens/inventory/AbstractFurnaceScreen
 		ARG 1 keyCode
 		ARG 2 scanCode
 		ARG 3 modifiers
+	METHOD lambda$init$0 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 imageButton
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/AbstractFurnaceScreen.mapping]:
```suggestion
		ARG 1 recipeButton
```

File: data/net/minecraft/client/gui/screens/inventory/BookEditScreen.mapping [hunk 3]
```diff
@@ -75,6 +91,8 @@ CLASS net/minecraft/client/gui/screens/inventory/BookEditScreen
 		ARG 3 modifiers
 	METHOD updateButtonVisibility ()V
 		COMMENT Sets visibility for book buttons
+	METHOD updateLocalCopy (Z)V
+		ARG 1 signed
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/BookEditScreen.mapping]: `signed` suggests the current state where this is only true if the book is being signed. `saveChanges` param should also be updated. 
```suggestion
		ARG 1 sign
```

File: data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping [hunk 4]
```diff
@@ -25,18 +39,32 @@ CLASS net/minecraft/client/gui/screens/inventory/BookViewScreen
...
 	CLASS BookAccess
+		METHOD fromItem (Lnet/minecraft/world/item/ItemStack;)Lnet/minecraft/client/gui/screens/inventory/BookViewScreen$BookAccess;
+			ARG 0 stack
 		METHOD getPageCount ()I
 			COMMENT Returns the size of the book
+		METHOD getPageRaw (I)Lnet/minecraft/network/chat/FormattedText;
+			ARG 1 index
 	CLASS WrittenBookAccess
+		METHOD <init> (Lnet/minecraft/world/item/ItemStack;)V
+			ARG 1 stack
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping]:
```suggestion
			ARG 1 writtenBookStack
```

File: data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping [hunk 5]
```diff
@@ -25,18 +39,32 @@ CLASS net/minecraft/client/gui/screens/inventory/BookViewScreen
...
 		METHOD getPageCount ()I
 			COMMENT Returns the size of the book
+		METHOD readPages (Lnet/minecraft/world/item/ItemStack;)Ljava/util/List;
+			ARG 0 stack
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping]:
```suggestion
			ARG 0 writtenBookStack
```

File: data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping [hunk 6]
```diff
@@ -25,18 +39,32 @@ CLASS net/minecraft/client/gui/screens/inventory/BookViewScreen
...
 	CLASS WritableBookAccess
+		METHOD <init> (Lnet/minecraft/world/item/ItemStack;)V
+			ARG 1 stack
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping]:
```suggestion
			ARG 1 writableBookStack
```

File: data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping [hunk 7]
```diff
@@ -25,18 +39,32 @@ CLASS net/minecraft/client/gui/screens/inventory/BookViewScreen
...
 		METHOD getPageCount ()I
 			COMMENT Returns the size of the book
+		METHOD readPages (Lnet/minecraft/world/item/ItemStack;)Ljava/util/List;
+			ARG 0 stack
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/BookViewScreen.mapping]:
```suggestion
			ARG 0 writableBookStack
```

File: data/net/minecraft/client/gui/screens/inventory/CraftingScreen.mapping [hunk 8]
```diff
@@ -16,6 +16,8 @@ CLASS net/minecraft/client/gui/screens/inventory/CraftingScreen
 		ARG 4 height
 		ARG 5 mouseX
 		ARG 7 mouseY
+	METHOD lambda$init$0 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 imageButton
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/CraftingScreen.mapping]:
```suggestion
		ARG 1 recipeButton
```

File: data/net/minecraft/client/gui/screens/inventory/InventoryScreen.mapping [hunk 9]
```diff
@@ -18,6 +18,8 @@ CLASS net/minecraft/client/gui/screens/inventory/InventoryScreen
 		ARG 4 height
 		ARG 5 mouseX
 		ARG 7 mouseY
+	METHOD lambda$init$0 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 imageButton
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/InventoryScreen.mapping]:
```suggestion
		ARG 1 recipeButton
```

File: data/net/minecraft/client/gui/screens/inventory/JigsawBlockEditScreen.mapping [hunk 10]
```diff
@@ -1,10 +1,21 @@
...
-		ARG 1 jigsawBlockEntity
+		ARG 1 jigsawEntity
 	METHOD keyPressed (III)Z
 		ARG 1 keyCode
 		ARG 2 scanCode
 		ARG 3 modifiers
+	METHOD lambda$init$3 (Lnet/minecraft/client/gui/components/CycleButton;Lnet/minecraft/world/level/block/entity/JigsawBlockEntity$JointType;)V
+		ARG 2 joint
+	METHOD lambda$init$4 (Lnet/minecraft/client/gui/components/CycleButton;Ljava/lang/Boolean;)V
+		ARG 1 cycleButton
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/JigsawBlockEditScreen.mapping]:
```suggestion
		ARG 1 keepJigsawsButton
```
(or just `button`)

File: data/net/minecraft/client/gui/screens/inventory/StructureBlockEditScreen.mapping [hunk 11]
```diff
@@ -1,10 +1,33 @@
...
+	METHOD lambda$init$2 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 button
+	METHOD lambda$init$3 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 button
+	METHOD lambda$init$4 (Lnet/minecraft/client/gui/components/Button;)V
+		ARG 1 button
+	METHOD lambda$init$5 (Lnet/minecraft/world/level/block/state/properties/StructureMode;)Lnet/minecraft/network/chat/Component;
+		ARG 0 structureMode
+	METHOD lambda$init$6 (Lnet/minecraft/client/gui/components/CycleButton;Lnet/minecraft/world/level/block/state/properties/StructureMode;)V
+		ARG 1 cycleButton
```
Comment (Harleyoc1) [inline data/net/minecraft/client/gui/screens/inventory/StructureBlockEditScreen.mapping]:
```suggestion
		ARG 1 modeButton
```
(or just `button`)

**Review by Harleyoc1 — CHANGES_REQUESTED** [review]

**Review by Harleyoc1 — APPROVED** [review]

**Review by alcatrazEscapee — APPROVED** [review]

**Pull request merged**